cmd/tether/main_linux.go

```diff
@@ -156,3 +151,22 @@ func halt() {
 	syscall.Sync()
 	syscall.Reboot(syscall.LINUX_REBOOT_CMD_POWER_OFF)
 }
+
+func startSignalHandler() error {
```

Comment:
This this should be in `lib/tether/tether_linx.go` - given it's the `lib/tether` code that uses the config it makes sense to have the reload trigger there.